docs/advanced/developer-guide.md

```diff
@@ -166,41 +166,49 @@ Command line flags of nfd-master:
 ```bash
 $ docker run --rm ${NFD_CONTAINER_IMAGE} nfd-master --help
 ...
-Usage:
-  nfd-master [--prune] [--no-publish] [--label-whitelist=<pattern>] [--port=<port>]
+  Usage:
+  nfd-worker [--no-publish] [--sources=<sources>] [--label-whitelist=<pattern>]
```

Comment:
I think there is an issue here as it is the nfd-worker help message instead of the nfd-master help message

Reply:
Argh 🤦  Thanks for noticing this! Fixed